extensions/amp-a4a/0.1/test/test-refresh-manager.js

```diff
@@ -54,9 +55,15 @@ describe('refresh-manager', () => {
     sandbox.restore();
   });
 
+  it('should get null from slot', () => {
```

Comment:
How about "should get null refreshInterval"?

Reply:
Done.